requirements.typ

```diff
@@ -1,4 +1,4 @@
 #import "/src/deps.typ"
 #import "@preview/tidy:0.1.0"
-#import "@preview/tidy:0.2.0"
+#import "@preview/tidy:0.3.0"
```

Comment:
Why do we import tidy twice? And given the current progress on the docs I don't think we'll need to update tidy.

Reply:
Because some parts of the docs only work with tidy 0.1.0.
As long as we do not have full online docs, we should update the manual.

Reply:
I think if you're importing it twice, the second `tidy` module shadows the first, no?